modules/upgrade/pages/upgrade-script-example.adoc

```diff
@@ -179,34 +230,25 @@ Use this script as the basis for an upgrade from SLE{nbsp}15 to SLE{nbsp}15.1.
         </source>
       </script>
     </chroot-scripts>
+    <init-scripts config:type="list">
+      $SNIPPET('spacewalk/minion_script')
+<!-- when client is traditional registered, please use this snippet
+      $SNIPPET('spacewalk/sles_register_script')
+-->
+    </init-scripts>
   </scripts>
 </profile>
 ----
 
 
 
-== Upgrading an SAP Client from SLE{nbsp}15 to SLE{nbsp}15.1
+== Upgrading an SAP Client from SLE{nbsp}12{nbsp}SPn to SLE{nbsp}15{nbsp}SP1
 
-Use this script as the basis for an upgrade from SLE{nbsp}15 to SLE{nbsp}15.1.
+Use this script as the basis for an upgrade from SLE{nbsp}12{nbsp}SPn to SLE{nbsp}15.1.
+Please check the official Product documentation to find the supported upgrade paths.
```

Comment:
```suggestion
Please check the official product documentation to find the supported upgrade paths.
```

Reply:
The other place with this sentence was suggested to be:

```suggestion
Check the official product documentation to find the supported upgrade paths.
```